openQA.spec

```diff
@@ -17,7 +17,7 @@
 
 
 # can't use linebreaks here!
-%define openqa_services openqa-webui.service openqa-gru.service openqa-websockets.service openqa-scheduler.service openqa-enqueue-audit-event-cleanup.service openqa-enqueue-audit-event-cleanup.timer openqa-enqueue-asset-and-result-cleanup.service openqa-enqueue-asset-and-result-cleanup.timer
+%define openqa_services openqa-webui.service openqa-gru.service openqa-websockets.service openqa-scheduler.service openqa-enqueue-audit-event-cleanup.service openqa-enqueue-audit-event-cleanup.timer openqa-enqueue-asset-and-result-cleanup.service openqa-enqueue-asset-and-result-cleanup.timer openqa-enqueue-limit_bugs.service openqa-enqueue-limit_bugs.timer
```

Comment:
Why not call it `openqa-enqueue-bug-cleanup.service/timer` for consistency with the other files?